docs/development.md

```diff
@@ -0,0 +1,76 @@
+# Development
+
+This documentation points to help developers that wants to contribute on the project, and wants to know how to set up it locally for development.
+
+## Requirements
+
+It's recommended used the version of node that is on the [.nvmrc](../.nvmrc) file, if you are using nvm would be enough if you switch to that version before installing dependencies.
```

Comment:
Maybe a phrase like "or download/set the version yourself" or similar for people that don't use nvm